src/index.html

```diff
@@ -22,6 +22,366 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <picture class="logo-container">
+        <a
+          href="/"
+          class="logoimage"
+        >
+          <img
+            src="./images/logo.png"
+            alt="MOYO Logo"
+          />
+        </a>
+      </picture>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li>
+            <div id="apple">
+              <a
+                href="#apple"
+                class="menu-item is-active"
+              >
+                Apple
+              </a>
+            </div>
+          </li>
+          <li>
+            <div id="samsung">
+              <a
+                href="#samsung"
+                class="menu-item"
+              >
+                Samsung
+              </a>
+            </div>
+          </li>
+          <li>
+            <div id="smartphones">
+              <a
+                href="#smartphones"
+                class="menu-item"
+              >
+                Smartphones
+              </a>
+            </div>
+          </li>
+          <li>
+            <div id="laptops">
+              <a
+                href="#laptops"
+                class="menu-item"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </div>
+          </li>
+          <li>
+            <div id="gadgets">
+              <a
+                href="#gadgets"
+                class="menu-item"
+              >
+                Gadgets
+              </a>
+            </div>
+          </li>
+          <li>
+            <div id="tablets">
+              <a
+                href="#tablets"
+                class="menu-item"
+                >
+                Tablets
+              </a>
+            </div>
+          </li>
+          <li>
+            <div id="photo">
+              <a
+                href="#photo"
+                class="menu-item"
+              >
+                Photo
+              </a>
+            </div>
+          </li>
+          <li>
+            <div id="video">
+              <a
+                href="#video"
+                class="menu-item"
+              >
+                Video
+              </a>
+            </div>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+    <section class="product" data-qa="card">
```

Comment:
The `data-qa` attribute is applied to the section with class `product`. Ensure that this attribute is applied consistently across all product sections if required by the task.